pkg/oldapis/direct.csi.min.io/v1alpha1/types.go

```diff
@@ -0,0 +1,252 @@
+// This file is part of MinIO DirectPV
```

Comment:
we deal with only types >= v1beta3. The utils from the pkgs < v1beta3 can be removed